### ISSUE TITLE
How to determine mailbox access?

### ISSUE THREAD

**Issue by tylerlittlefield** [Azure/Microsoft365R#211]
How can I determine which mailboxes a user has access to? Or in the opposite direction, which users have access to a specific mailbox?

**Comment by hongooi73** [comment]
I don't think you can discover this with this package. You can check the [Graph docs](https://docs.microsoft.com/en-us/graph/api/overview?view=graph-rest-beta) if a suitable API function is available.
